### DISCUSSION TITLE
Welcome to documenter Discussions!

### DISCUSSION THREAD

**Discussion by KutnerUri** [teambit/documenter#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KutnerUri** [comment]
cool this is basically 'issues' with a reddit like sort

**Reply by KutnerUri** [reply]
reply test

**Reply by KutnerUri** [reply]
that's pretty neat

**Reply by giteden** [reply]
hey
